buildworker.rst

```diff
@@ -80,42 +80,40 @@ For Mac:
     * Create a buildbot user using the OS/X control panel user admin.  It
       should be a "standard" user.
     * Log in as the buildbot user.
-    * Install either the Python 2.7 bundle from python.org [#]_, or pip.
-    * Open a terminal window.
-    * Execute ``pip install buildbot-slave``.
+    * Install the buildbot worker, either from python.org [#]_ or by running ``pip install buildbot-worker``.
```

Comment:
This isn't quite right; `buildbot-worker` is not available from python.org.  I'd say just take out everything between `worker,` and `by`.